script/make_utils/jupyter.sh

```diff
@@ -13,9 +13,10 @@ print_time_execution() {
 
 WHAT_TO_DO="open"
 
-# Create a list of notebooks with long execution times in order not to consider them when refreshing
-# all notebooks at the same time.
-LONG_EXECUTION_TIMES_NOTEBOOKS=()
+# Create a list of notebooks to skip, usually because of their long execution time.
+# Deployment notebook is currently failing
+# FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4064
+NOTEBOOKS_TO_SKIP=("docs/advanced_examples/Deployment.ipynb")
```

Comment:
the notebook does not work (it's been quite some time now) so we're skipping it for now

Reply:
it will be removed in Jordan's PR